src/main/webapp/populateData.js

```diff
@@ -89,6 +95,7 @@ function populateTasks() {
       });
 }
 
+
```

Comment:
Only one blank line is needed between methods

Reply:
Good catch, thanks!

Reply:
`RESOLVED`